### PR TITLE
Add GetOffsetFromWorldCoords and GetOffsetInWorldCoords to Camera and…

### PR DESCRIPTION
**Pull request by zorg93**
… GameplayCamera

### CODE REVIEW

**Comment by zorg93** [comment]
None of the Scaleform changes break current 2.X scripts, but obselete functions should be removed for 3.X

**Comment by crosire** [comment]
Should we maybe reference count the scaleform instances and load/unload them in the constructor/destructor, so the user does not have to bother?

**Comment by zorg93** [comment]
Load isnt the best name for that method. When the load function returns the scaleform isnt actually loaded in the sense that has_scaleform_movie_loaded will still return false. Its more a case of seeing if the movie is valid, then assigning a handle to it. The reason for making the unload method public is in the case of a script termination the destructor wont be called

**Comment by Fireboyd78** [comment]
Was this supposed to be merged into the `dev` branch? I noticed that `ScaleformMovie.Load()` was removed instead of being made obsolete.

**Comment by crosire** [comment]
Uhm. But it wasn't removed and is marked obsolete?

**Comment by Fireboyd78** [comment]
@crosire Nope, the only one that was marked as obsolete is the `ScaleformMovie.Load(int Handle)` method. There was one that had a `void` constructor.

**Comment by zorg93** [comment]
the void constructor was internal and that was handled

File: source/scripting/Scaleform.cpp [hunk 1]
```diff
@@ -27,19 +28,9 @@ namespace GTA
 	{
 		return Native::Function::Call<bool>(Native::Hash::HAS_SCALEFORM_MOVIE_LOADED, _handle);
 	}
-
-	bool Scaleform::Load()
```
Comment (Fireboyd78) [inline source/scripting/Scaleform.cpp]: @crosire See, this was removed.

**Comment by Fireboyd78** [comment]
@zorg93 Oh, ok. Thanks for clearing that up.

**Comment by zorg93** [comment]
I tried running the modifed version with a script compiled against old methods and it still worked fine

**Comment by Fireboyd78** [comment]
@zorg93 Yeah my apologies, I didn't notice it was an internal method when I was talking about it. I'll try to pay more attention next time =P

**Pull request merged**